CHANGELOG.md

```diff
@@ -4,10 +4,14 @@ Notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 Kubeclient release versioning follows [SemVer](https://semver.org/).
 
-## 4.1.1 — 2018-12-17
+## 4.1.1 — 2018-12-26
```

Comment:
can't have 4.1.1 twice.  Make this `## Unreleased` or `## 4.2.1`, I'll set the version & date in a later PR for the release version bump.

hmm, you're still based on v4.1.z branch.  you should merge/rebase fresh master, put the new version on top, before 4.2.0 (it'll become 4.2.1).
(unless you have a need for this fix to also be released on 4.1 branch?)

Reply:
fog-kubevirt has the following dependency on kubeclient: "kubeclient", "~> 4.1.0"
So I'd like it to be included and released for 4.1.z (more than have it on 4.2.0 in terms on necessity)

Reply:
@cben PR for master is here: https://github.com/abonas/kubeclient/pull/383